Scripts/script-TopMaliciousRatioIndicators.yml

```diff
@@ -0,0 +1,39 @@
+commonfields:
+  id: TopMaliciousRatioIndicators
+  version: -1
+fromversion: "4.0.0"
+name: TopMaliciousRatioIndicators
+script: |
+  from datetime import datetime, timedelta
+  import json, random
+
+  MAX_INDICATORS = 10000
```

Comment:
make this an argument

Reply:
It's for dashboard use - we can't really set custom argument. we still want to add this?

Reply:
if this is a dashboard script, aren't you missing:
```
tags:
- widget
```
?